helix-lsp-types/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "helix-lsp-types"
+version = "0.95.1"
+authors = ["Markus Westerlind <marwes91@gmail.com>", "Bruno Medeiros <bruno.do.medeiros@gmail.com>"]
```

Comment:
I think we should probably add ourself to the authors list. If we publish it it would be weird to publish in someone else's name. Having both us and the original authors here feels like a good balance between giving credit and reflecting current realities